consensus/polybft/extra_test.go

```diff
@@ -122,7 +120,7 @@ func TestExtra_UnmarshalRLPWith_NegativeCases(t *testing.T) {
 
 		extra := &Extra{}
 		ar := &fastrlp.Arena{}
-		require.ErrorContains(t, extra.UnmarshalRLPWith(ar.NewArray()), "incorrect elements count to decode Extra, expected 5 but found 0")
+		require.ErrorContains(t, extra.UnmarshalRLPWith(ar.NewArray()), "incorrect elements count to decode Extra, expected 6 but found 0")
```

Comment:
This assert isn't correct, test is failing.

We should require this error message instead:
"incorrect elements count to decode Extra, expected 5 but found 0"

Reply:
fixed here: https://github.com/0xPolygon/polygon-edge/pull/894/commits/a1c83a536745353298e74dd3c360efd8be8f08ca